src/develop/pixelpipe_cache.c

```diff
@@ -220,20 +220,20 @@ gboolean dt_dev_pixelpipe_cache_available(
 
   dt_dev_pixelpipe_cache_t *cache = &(pipe->cache);
   // search for hash in cache and make the sizes are identical
-  for(int k = 0; k < cache->entries; k++)
+  for(int k = DT_PIPECACHE_MIN; k < cache->entries; k++)
```

Comment:
This one was 0, is the change to DT_PIPECACHE_MIN expected? Was this a left over from original PR?

Reply:
Yes a leftover and safe this way

Reply:
Good!